src/NuGetForUnity/Editor/Configuration/NugetConfigFile.cs

```diff
@@ -88,6 +86,12 @@ public class NugetConfigFile
         /// </summary>
         public bool Verbose { get; set; }
 
+        /// <summary>
+        ///     Gets or sets a value indicating whether to skip installing dependencies and checking for pre-imported Unity libs
+        ///     while auto-restoring.
+        /// </summary>
+        public bool SlimRestore { get; set; }
```

Comment:
```suggestion
        public bool SlimRestore { get; set; } = true;
```
To switch it on 'by default' if someone updates to the new version of NuGetForUnity. I think it is valid to enable it by default if we add a not inside the Release Notes. Or what do you think @igor84, @popara96?

Reply:
We thought about it. We are not sure what the best option is but we concluded the safest is to leave this false in existing projects so they continue working as they did and users having to manually enable it in old projects if they wish to change the behavior, while we default to true only for new projects.